ClearKeep/Scenes/PeerCall/Views/SingleVoiceCallInfoView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SingleVoiceCallInfoView.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 25/07/2022.
+//
+
+import SwiftUI
+import CommonUI
+
+struct SingleVoiceCallInfoView: View {
+	@ObservedObject var viewModel: CallViewModel
+	
+	func image(withName: String?) -> Image? {
+		guard let name = withName, !name.isEmpty else {
+			return nil
+		}
+		return Image(name)
+	}
+	
+	var body: some View {
+		GeometryReader { reader in
+			VStack {
+				Spacer()
+				
+				VStack(spacing: 32) {
+					Spacer()
+						.frame(height: 60)
+					
+					if viewModel.callType == .audio, viewModel.callStatus != .answered, !viewModel.getStatusMessage().isEmpty {
+						Text(viewModel.getStatusMessage())
+							.font(AppTheme.shared.fontSet.font(style: .input2))
+							.foregroundColor(AppTheme.shared.colorSet.grey5)
+					}
+					AvatarDefault(.constant(viewModel.getUserName()), imageUrl: viewModel.callBox?.avatar ?? "")
+						.font(Font.system(size: 72))
+						.frame(width: 160, height: 160)
+					
+					VStack(spacing: 8) {
+						Text(viewModel.getUserName())
+							.font(AppTheme.shared.fontSet.font(style: .display2))
+							.foregroundColor(AppTheme.shared.colorSet.offWhite)
+							.lineLimit(2)
+							.padding(.horizontal, 16)
+							.layoutPriority(1)
+						
+						if viewModel.callType == .audio, viewModel.callStatus == .answered {
+							Text(viewModel.timeCall)
+								.font(AppTheme.shared.fontSet.font(style: .heading3))
+								.foregroundColor(AppTheme.shared.colorSet.offWhite)
+						}
+					}
+					
+					Spacer()
+					
+					Spacer()
+						.frame(height: reader.size.height/2)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/PeerCall/Views/SingleVoiceCallInfoView.swift#L57"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>SingleVoiceCallInfoView.swift:57</code></td>
    </tr>
  </tbody>
</table>